assets/donate.css

```diff
@@ -0,0 +1,136 @@
+:root {
+  --gray: #798890;
+  --light-blue: #0f81e8;
+}
+
+.payments-titles {
+    display: flex;
+    align-items: center;
+    justify-content: space-around;
+    margin-bottom: 1em; /* 16px */
+  }
+
+  .payments-title {
+    border-bottom: 0.1875rem solid;  /* 3px */
+    border-bottom-color: #798890;
+    border-bottom-color: var(--gray);
+    width: 100%;
+    margin-right: 0.125rem; /* 2px */
+    text-align: center;
+    cursor: pointer;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .payments-title-active {
+    border-bottom-color: #0f81e8;
+    border-bottom-color: var(--light-blue);
+  }
+
+  .card-title span {
+    margin-right: 0.3em;  /* 4.8px */
+    font-weight: bold;
+    color: #798890;
+    color: var(--gray);
+    font-size: small;
+  }
+
+  #ak-continue-button,
+  .ak-submit-button {
+    width: 100%;
+  }
+
+  .paypal-only {
+    border-bottom: none;
+    cursor: auto;
+  }
+
+  .button:focus {
+    outline: none;
+  }
+
+  .payment-title-button {
+    border: 0.125rem solid #798890;
+    border: 0.125rem solid var(--gray);  /* 2px border */
+    border-radius: 0.375rem; /* 6px */
+    margin-right: 0.75rem; /* 12px */
+    margin: 0 0.375rem; /* margin: 0 6px */
+    padding: 0.1875rem 0; /* padding: 3px 0 */
```

Comment:
is there a reason why we have the pixel values in comments?

Reply:
We generally include them in comments because it's helpful for us to see the pixel conversion. I can remove if it's not useful, though!